src/main/java/sirpotato/SirPotato.java

```diff
@@ -0,0 +1,71 @@
+package sirpotato;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.LocalDate;
+
+import java.util.ArrayList; 
+import java.util.Scanner;  
```

Comment:
I like how you left an empty line between each group of imports and keep the order of imports consistent so it's easier to browse the list.